apps/frontend-pwa/src/pages/course/[courseId]/docs/appSetup.tsx

```diff
@@ -7,6 +7,7 @@ function AppSetup() {
     <DocsLayout>
       {(courseInformation) => (
         <Markdown
+          withProse
```

Comment:
new option that uses <Prose> instead of <div> as a base element for the markdown component (which is a popular use case for markdown text)